lib/src/settings.rs

```diff
@@ -44,14 +44,14 @@ pub struct RepoSettings {
 
 #[derive(Debug, Clone)]
 pub struct GitSettings {
-    pub auto_local_branch: bool,
+    pub auto_local_bookmark: bool,
     pub abandon_unreachable_commits: bool,
 }
 
 impl GitSettings {
     pub fn from_config(config: &config::Config) -> Self {
         GitSettings {
-            auto_local_branch: config.get_bool("git.auto-local-branch").unwrap_or(false),
+            auto_local_bookmark: config.get_bool("git.auto-local-bookmark").unwrap_or(false),
```

Comment:
This is a breaking change (so needs changelog entry.) Might be okay to leave it unchanged since "branch" is a valid term in Git context.

Reply:
reverted.